modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -201,7 +201,9 @@ function format_wifirate(rate) {
 	    mhz = rate.mhz, nss = rate.nss,
 	    mcs = rate.mcs, sgi = rate.short_gi,
 	    he = rate.he, he_gi = rate.he_gi,
-	    he_dcm = rate.he_dcm;
+	    he_dcm = rate.he_dcm,
+	    eht = rate.eht, eht_gi = rate.eht_gi,
+	    eht_dcm = rate.eht_dcm;
```

Comment:
Minor nitpick - I tested this code on an 802.11ac device and it runs OK. In case these properties don't exist, it's good practice to provide default values when trying to access possibly non-existent values/properties. You can supply other defaults you deem appropriate if you think 0 is no good, but I chose 0 which is fine if `eht` is false:
```js
	    eht = rate?.eht ?? false, eht_gi = rate?.eht_gi ?? 0,
	    eht_dcm = rate?.eht_dcm ?? 0;
```

Reply:
I haven't chosen to initialise the eht variables has none of the others (ht and he) are being initialised either

Reply:
Not yet ;) 

Could you put it in? Then the next pass over this file should amend those also. I'll look at that.